### PR TITLE
Newcomers Guide: Add issue assignment warning

### PR DESCRIPTION
**Pull request by sils**

### CODE REVIEW

File: docs/Developers/Newcomers_Guide.rst [hunk 1]
```diff
@@ -3,6 +3,11 @@
 Welcome to the Newcomers Guide!
 ===============================
 
+**DO NOT WORK ON ANY ISSUE WITHOUT ASSIGNMENT!** If you do, someone else might
+work on it as well and we might have no choice but reject one of your Pull
+Requests - we hate it if anyone wastes his time. For your own sake, please
```
Comment (sims1253) [inline docs/Developers/Newcomers_Guide.rst]: their time

**Comment by sims1253** [comment]
ack 88f126e

**Comment by sims1253** [comment]
ack 88f126e

**Comment by sils** [comment]
ack 88f126e

**Comment by sims1253** [comment]
@rultor merge

**Pull request merged**